flashback: add alert msg in flashback statement (#13722)
pingcap/docs: sql-statements/sql-statement-flashback-to-timestamp.md

</CustomContent>

> **Warning:**
>

> > When you use this feature in TiDB v7.1.0, some Regions might not be restored after the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).

Reviewer: ```suggestion
> When you use this feature in TiDB v7.1.0, some Regions might not be finished after the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).
```

> > **Warning:**
> >
> > When you use this feature in TiDB v7.1.0, some Regions might not be restored after the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).
> >
> > If you have encountered this issue, you can use the [TiDB snapshot backup and restore](/br/br-snapshot-guide.md) feature to restore data.

Author: ```suggestion
<CustomContent platform="tidb">

> **Warning:**
>
> When you use this feature in TiDB v7.1.0, some Regions might not be restored after the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).
>
> If you have encountered this issue, you can use the [TiDB snapshot backup and restore](/br/br-snapshot-guide.md) feature to restore data.

</CustomContent>
```

</CustomContent>

<CustomContent platform="tidb">

> **Warning:**
>

> > When you use this feature in TiDB v7.1.0, some Regions might not be finished after the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).

Author: ```suggestion
> When you use this feature in TiDB v7.1.0, some Regions might remain in the FLASHBACK process even after the completion of the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).
```
@Defined2014 PTAL
